<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolManager</class>
 <widget class="QWidget" name="ToolManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>工具栏</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QToolButton" name="Circle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="toolTip">
         <string>圆形标注</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="statusTip">
         <string>新建圆形标注</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/CIrcle</normaloff>:/icon/CIrcle</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="Draw">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>分割标注画笔</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="statusTip">
         <string>分割标注画笔</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/Brush</normaloff>:/icon/Brush</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="Square">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>正方标注</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="statusTip">
         <string>新建正方标注</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/Rect</normaloff>:/icon/Rect</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="Mouse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>鼠标箭头</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="statusTip">
         <string>鼠标箭头</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/Mouse</normaloff>:/icon/Mouse</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="Erase">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>分割标注擦除</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="statusTip">
         <string>分割标注擦除</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/Rubber</normaloff>:/icon/Rubber</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="Brush">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>分割标注</string>
        </property>
        <property name="toolTipDuration">
         <number>10</number>
        </property>
        <property name="statusTip">
         <string>新建分割标注</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/DrawingPad</normaloff>:/icon/DrawingPad</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QToolButton" name="ZoomIn">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>视图放大</string>
        </property>
        <property name="statusTip">
         <string>视图放大</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/ZoomIn</normaloff>:/icon/ZoomIn</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="ZoomOut">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>视图缩小</string>
        </property>
        <property name="statusTip">
         <string>视图缩小</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/ZoomOut</normaloff>:/icon/ZoomOut</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="Lens">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="toolTip">
         <string>放大镜</string>
        </property>
        <property name="statusTip">
         <string>放大镜</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
        <property name="icon">
         <iconset resource="rsc.qrc">
          <normaloff>:/icon/Lens</normaloff>:/icon/Lens</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
